.github/workflows/test.yml

```diff
@@ -17,7 +17,7 @@ concurrency:
 env:
   GITHUB_REF: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.ref || github.ref }}
   IS_DEV: ${{ github.event_name != 'pull_request' && github.ref == 'refs/heads/development' }}
-
+  ORYX_FEATURE_VERSION: latest
```

Comment:
Was it added by purpose?

Reply:
Yes, to overpass e2e tests, otherwise price mode would not exist